src/components/checkbox/index.tsx

```diff
@@ -186,14 +201,18 @@ class Checkbox extends Component<CheckboxProps, CheckboxState> {
       >
         {
           <Animated.View
-            style={[this.styles.container, {backgroundColor: this.getColor()}, {opacity: this.animationStyle.opacity}]}
+            style={[
+              this.styles.container,
+              {backgroundColor: outline ? 'transparent' : disabled ? DEFAULT_DISABLED_COLOR : this.getColor()},
+              {opacity: this.animationStyle.opacity}
+            ]}
           >
             <Animated.Image
               style={[
                 this.styles.selectedIcon,
-                color && {tintColor: iconColor},
+                color && {tintColor: outline ? this.getColor() : DEFAULT_ICON_COLOR},
```

Comment:
There's an issue here. 
You ignore the `iconColor` prop. 
If a checkbox starts disabled and then become enabled in runtime. 
The checkmark icon color will stay grey 

Also, with when passing the `outline` flag, `iconColor` is being ignored. 

I suggest moving the logic of the icon color to a separate  function (getIconColor) and apply in runtime. 
setting it in `createStyles` will not take effect in runtime..

Reply:
@ethanshar thanks for the quick feedback!  I'll add this soon. 

I also noticed a bug with 'outline - no color - disabled` seen in the screenshot above which I think is caused by the runtime scenario you explained.   I'll do another round of cleaning it up today and get back to you.

Reply:
@ethanshar this should be fixed now with the new commit!  

I also took the liberty of creating DEFAULT_BORDER_WIDTH and DEFAULT_BORDER_RADIUS constants and included them with the rest of the constants for this page.  I hope this is ok!

Reply:
@ethanshar  what is the process here?  Should I resolve this or leave it for you to review then close?

Reply:
Hi,
Sorry for the late response. 
There's still small issue.
See my comment.